web_translate_dialog/models/base.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2019 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl)
+from odoo import models, api
+
+
+class BaseModel(models.BaseModel):
+
+    _inherit = 'base'
```

Comment:
No need for this:

https://github.com/OCA/OCB/blob/12.0/odoo/models.py#L434

Reply:
that's added only when current name is not `base`. In any case: explicit it's better than implicit :wink: